docs/pbc/all/tax-management/202311.0/base-shop/third-party-integrations/vertex/install-vertex.md

```diff
@@ -136,25 +140,67 @@ use Spryker\Zed\TaxApp\Communication\Plugin\Calculation\TaxAppCalculationPlugin;
 // ...
 ```
 
-{% info_block infoBox "Performance improvement" %}
+{% info_block infoBox "Updating from previous version" %}
 
-Spryker has its own [Taxes](/docs/pbc/all/tax-management/{{page.version}}/base-shop/tax-feature-overview.html) feature, which comes pre-installed in the Checkout through the Calculation module. To enhance performance when using an external Tax calculation provider, we recommend disabling the following plugins:
+Please pay attention to the position of TaxAppCalculationPlugin in the list of plugins.
```

Comment:
please add between what exactly plugins we expect to see 
(I understand that it's visible in the code snippet, but can clarify more)